cmd/ocsp-responder/main.go

```diff
@@ -151,7 +151,7 @@ func main() {
 
 		// Configure HTTP
 		m := http.NewServeMux()
-		m.Handle(c.OCSPResponder.Path, cfocsp.Responder{Source: src})
+		m.Handle(c.OCSPResponder.Path, cfsslOCSP.Responder{Source: src})
```

Comment:
why this change? Typically,  capitals are not used in package names and this one was already well named.